frontend/app/views/spree/shared/_nav_bar.html.erb

```diff
@@ -1,4 +1,4 @@
-<ul id="nav-bar" class="nav align-items-center d-flex flex-nowrap justify-content-end navbar-right">
+<ul id="nav-bar" class="nav align-items-center d-flex flex-nowrap justify-content-end navbar-right" data-turbolinks-permanent>
```

Comment:
@MatthewKennedy let's not have it here, with Turbo(Drive) and language switching this will not refresh the nav